runners/flink/1.10/build.gradle

```diff
@@ -17,6 +17,13 @@
  */
 
 def basePath = '..'
+def jackson_annotations_version = "2.10.2"
+def flink_clients_version = "1.10.1"
+def jackson_databind_verisioon = "2.10.2"
```

Comment:
If the version is different between of Flink 1.10 then it could be listed here, but needs a comment to explain it.

Reply:
fixed it by library.java.jackson_databind. Now we no longer need version declaration.